use std::error::Error;
use std::fmt::{Debug, Display};

fn main() -> Result<(), Box<dyn Error>> {
    Ok(())
}
